R/fct_SQLite.R

```diff
@@ -498,28 +483,26 @@ db_get_query <- function(
 #'   ) |>
 #'    dplyr::as_tibble()
 #'   DBI::dbWriteTable(con, "all_review_data", review_data)
-#'   db_get_review(temp_path, subject = "Test_name", form = "Test_group")
+#'   db_get_review(temp_path, ids = 1L)
 #' })
 #' 
 db_get_review <- function(
     db_path, 
-    subject = review_row$subject_id, 
-    form = review_row$item_group,
-    db_table = "all_review_data",
-    slice_vars = c("timestamp", "edit_date_time"),
-    group_vars = c("subject_id", "event_name", "item_group",
-                   "form_repeat", "item_name")
+    ids = review_row$id,
```

Comment:
Substantial change that's open for discussion. Makes it harder to see which records are being grabbed in the tests.

Reply:
NM. I generalized the function.